pkg/volume/empty_dir/empty_dir.go

```diff
@@ -139,6 +147,7 @@ const (
 // These do not persist beyond the lifetime of a pod.
 type emptyDir struct {
 	podUID        types.UID
+	pod           *api.Pod
```

Comment:
I don't, and I've taken it out of the downstream patch for this.  I've been pretty much embroiled in prep for the OpenShift launch but once the dust settles I will update this PR.